Extensions/BBText/JsExtension.js

```diff
@@ -516,14 +516,27 @@ module.exports = {
         .getValue();
       this._pixiObject.textStyles.default.fontSize = `${fontSize}px`;
 
-      const fontFamily = this._associatedObject
+      const fontResourceName = this._associatedObject
         .getProperties(this.project)
         .get('fontFamily')
         .getValue();
-      this._pixiObject.textStyles.default.fontFamily = this._pixiResourcesLoader.getFontFamily(
-        this.project,
-        fontFamily
-      );
+
+      if (this._fontName !== fontResourceName) {
+        console.log('load font');
+
+        this._fontName = fontResourceName;
```

Comment:
We could even rename `this._fontName` to `this._fontResourceName` (to make extra clear what it represents). I'm ok with both